codedog/chains/pr_summary/base.py

```
@@ -1,18 +1,19 @@
+from langchain_core.callbacks.manager import (
     AsyncCallbackManagerForChainRun,
     CallbackManagerForChainRun,
 )
 from langchain.chains import LLMChain
 from langchain.chains.base import Chain
 from langchain.output_parsers import OutputFixingParser, PydanticOutputParser
-from langchain.schema import BaseOutputParser
+from langchain_core.output_parsers import BaseOutputParser
 from langchain_core.prompts import BasePromptTemplate
-from pydantic import Extra, Field
+from pydantic import Field, BaseModel, ConfigDict
```

Ruff (0.8.2)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)